vertx-sql-client/src/main/java/io/vertx/sqlclient/impl/SqlClientBase.java

```diff
@@ -53,6 +54,8 @@ public int appendQueryPlaceholder(StringBuilder queryBuilder, int index, int cur
     return current;
   }
 
+  protected abstract ContextInternal context();
```

Comment:
this seems unnecessary here because we use contexual promise interface instead?

Reply:
I need to check